client.go

```diff
@@ -25,3 +25,40 @@ func (c *client) metrics() (metrics, error) {
 	var res metrics
 	return res, c.rpcClient.Call(&res, "debug_metrics", true)
 }
+
+func (c *client) currentBlock() (string, error) {
+	var res string
+	return res, c.rpcClient.Call(&res, "eth_blockNumber")
+}
+
+func (c *client) currentBlockMetrics() (metrics, error) {
+	/* if not syncing we can at least get current block */
+	cblock, err := c.currentBlock()
+	/* and fake syncing metrics from it */
+	return metrics{
+		"currentBlock":  cblock,
+		"highestBlock":  cblock,
+		"startingBlock": cblock,
+		"knownStates":   "0",
+		"pulledStates":  "0",
+	}, err
+}
+
+func (c *client) syncingMetrics() (metrics, error) {
+	/* using interface{} to check type later */
+	var res interface{}
+	err := c.rpcClient.Call(&res, "eth_syncing")
+	/* syncing might be turned off, like in statusd */
+	if err != nil {
+		return c.currentBlockMetrics()
```

Comment:
I would do it like this:
- have `syncingMetrics()` as it is now,
- have `syncingMetricsFromCurrentBlock()` for the metrics based on the current block.

Reply:
Do you just want me to rename the method, or do you want some more refactoring?

Reply:
I was thinking about refactoring. But I think the current implementation is not that different and it's fine.